docs/AssembleRocker.md

```diff
@@ -23,7 +27,7 @@ Rocker subassembly complete.
 
 ![Rocker complete](images/Rocker05-Complete.jpg)
 
-Rocker subassembly plus bogie subassembly.
+Rocker subassembly plus bogie subassembly. The three wheel mounts should all be roughly in line with each other.
```

Comment:
This is incorrect - the center wheel actually sticks out further than the other two.

Reply:
I'll rephrase it then. The problem I had was on the initial assembly I did, I wound up with the center wheels actually splayed outwards because the Bogie Wheel part had gotten swapped side to side.

Reply:
Rocker subassembly plus bogie subassembly. The three wheel mounts should all be roughly in line with each other, with the center wheel extending slightly farther away from the body than the corner wheels. If the center wheel is splayed outward at an angle, the Bogie Wheel part has been swapped side to side.